edk2toollib/windows/telem/friendlynames.py

```diff
@@ -0,0 +1,47 @@
+# @file testdata.py
+# Dictionary for replacing guids with friendly names
+#
+# Copyright (c) Microsoft Corporation
+#
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+friendlynamedict = {"2dce8bb1-bdd7-450e-b9ad-9cf4ebd4f890": "CMC Notify Type",
```

Comment:
maybe capitalize this so it's clear it's a constant or turn it into a function that returns this?